data/Act.csv

```diff
@@ -702,9 +702,9 @@ A00930,P0056,AT29,PS00520,,195x,195x,Gu added comments and translated it into ve
 A00931,P0056,AT10,PS00521,,195x,195x,Gu was commissioned by Mao Zedong to take charge of the punctuation of the book.,SS00131,,2,,,2018-11-04,WH,2018-11-04,WH
 A00932,P0057,AT05,PS00522,,1966..1976,1966..1976,Xie made a joke with the reference of Empty Fort Strategy while doing labor work during the CR.,SS00131,,2,,,2018-11-04,WH,2018-11-04,WH
 A00933,P0263,AT27,PS00523,P0169,1966,1966,Mingxin’s teacher Wang Hanying encouraged him and lent when he said he wanted to try writing critiques and lent him the journals.,SS00207,,2,,,2018-11-04,WH,2018-11-04,WH
-A00934,P0120,AT05,AW00110,,1966.1976,1966.1976,He mentioned that this work considered not in line with official policy during the CR.,SS00030,13,2,,,2018-11-04,WH,2018-11-04,WH
-A00935,P0120,AT05,AW00111,,1966.1976,1966.1976,He mentioned that this work considered not in line with official policy during the CR.,SS00030,14,2,,,2018-11-04,WH,2018-11-04,WH
-A00936,P0120,AT05,AW00112,,1966.1976,1966.1976,He mentioned that this work considered not in line with official policy during the CR.,SS00030,14,2,,,2018-11-04,WH,2018-11-04,WH
+A00934,P0120,AT05,AW00110,,"1966,1976","1966,1976",He mentioned that this work considered not in line with official policy during the CR.,SS00030,13,2,,,2018-11-04,WH,2018-11-04,WH
```

Comment:
`,` is not the same as `.` see [the wiki](https://github.com/readchina/ReadingData/wiki/FAQ#we-dont-know-if-mr-x-was-born-in-1961-or-1962-do-i-need-a-new-field-for-that) probably another autoformatting error